src/components/AnnotationDesktop/SelectionURLState/SelectionURLState.tsx

```diff
@@ -32,6 +32,8 @@ export const SelectionURLState = (props: SelectionURLStateProps) => {
 
     if (selected.length > 0) {
       hash.set('selected', selected.join(','))
+    } else {
+      hash.delete('selected');
```

Comment:
Small issue I noticed: deselecting an annotation in the interface did not clear the URI hash.